qadence_protocols/mitigations/analog_zne.py

```diff
@@ -69,7 +69,7 @@ def mutate_params(block: AbstractBlock, stretch: float) -> AbstractBlock:
         converted_circuit = backend.circuit(stretched_circuit)
         noisy_density_matrices.append(
             # Contain a single experiment result for the stretch.
-            backend.run_dm(
+            backend.run(
```

Comment:
Not sure it is in scope with this PR :)

Reply:
No but it will need to be done with new qadence versions :)